viewer/packages/tools/src/Measurement/MeasurementManager.ts

```diff
@@ -115,6 +115,16 @@ export class MeasurementManager {
     this._line.updateLineColor(color);
   }
 
+  /**
+   * Hide/unhide all measurements
+   * @param enable
+   */
+  visible(enable: boolean): void {
```

Comment:
```suggestion
  showMeasurements(enable: boolean): void {
```
I was just thinking that this method should have a name similar to `showMeasurementLabels`. Or maybe it should be `setVisible(enable: boolean)` :thinking: Any opinions?

Reply:
We have similar function in `HTMLOverlayTool.visible()`, was following the same pattern